plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetReaderConfig.java

```diff
@@ -37,13 +37,11 @@ public class ParquetReaderConfig
 
     private ParquetReaderOptions options = new ParquetReaderOptions();
 
-    @Deprecated
```

Comment:
Was it intentional that the session property `parquet_ignore_statistics` was introduced only for Hive, but not for Iceberg (Delta Lake connector didn't exist at the time) in 1981cefb ?

cc @findepi

Reply:
it was originally added to Trino due to a bug in Trino which caused us to ingest corrupted statistics.